source/_integrations/tesla_fleet.markdown

```
@@ -37,30 +37,34 @@ ha_quality_scale: gold
-
-{% enddetails %}
-
 {% include integrations/config_flow.md %}
 
 ## Scopes
 
-When connecting your Tesla account to Home Assistant, you **must** select the `Vehicle Information` or `Energy Product Information` scope. It is recommended you select all scopes for full functionality.
+When connecting your Tesla account to Home Assistant, you **must** select the `Vehicle Information` or `Energy Product Information` scope. It is recommended you select all scopes for full functionality. The `Vehicle Location` scope was added in Home Assistant 2024.1, so any authorizations performed on previous releases that want this scope will need to be [modified](https://accounts.tesla.com/en_au/account-settings/security?tab=tpty-apps).
+
+## Pay per use
+	
```

Markdownlint (0.35.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)